Services/YoutubeService.php

```diff
@@ -456,13 +455,18 @@ public function getVideoMeta($yid)
     public function updatePlaylists(MultimediaObject $multimediaObject)
     {
         $youtube = $this->getYoutubeDocument($multimediaObject);
-        if (!isset($youtube)
-            || $youtube->getStatus() !== Youtube::STATUS_PUBLISHED) {
+        if (!isset($youtube) || $youtube->getStatus() !== Youtube::STATUS_PUBLISHED) {
```

Comment:
Here I actually agree with the change. There was no need to multi-line it, it is not THAT long. :+1: